jdaviz/configs/imviz/helper.py

```diff
@@ -114,9 +95,14 @@ def link_data(self, link_type='pixels', wcs_fallback_scheme='pixels', wcs_use_af
 
         .. warning::
 
-            Any markers added would be removed. You can add back the markers
-            manually using :meth:`add_markers`. During the markers removal,
-            pan/zoom will also reset.
+            Any markers added in default viewer would be removed automatically.
```

Comment:
Agree we are forced into weirdness here.  But two thoughts on what to me might make it a smidge more intuitive:
1. While it's a bit of a break to the independent-viewers idiom, I wonder if we should have the names be global to the app, either in the default viewer or in the helper? I.e., it would search through all of the viewers and look for a name match.  (If there are conflicts where the same name appears twice, just pick the first one but make a warning maybe?)
2. I don't understand why pan/zoom is reset.  Can that be fixed, or is it deep in the bowels of how the viewers are made?

Either/both of these can be follow-ons if they're going to be a lot of work though. Not necessary to hold this PR up for those.

Reply:
For your second question, it is #760 and that won't be fixed before release.

I'll look into polling viewers even though I decided against it initially.